src/azure-cli/azure/cli/command_modules/util/commands.py

```diff
@@ -17,3 +17,6 @@ def load_command_table(self, _):
 
     with self.command_group('demo', deprecate_info=g.deprecate(hide=True)) as g:
         g.custom_command('style', 'demo_style')
+
+    with self.command_group('debug', deprecate_info=g.deprecate(hide=True)) as g:
+        g.custom_command('echo', 'echo')
```

Comment:
`az debug echo` is designed to overcome the limitations of `--debug`:

- `az --debug '{"a":"b"}'` fails with a parser error and shows many irrelevant lines. `az debug echo` will succeed and only show the argument value.
- `az --debug @test.json` can't show the file content of the argument (only shows `'@test.json'`). `az debug echo` shows the actual file content.

`az debug echo` is hidden (not user-visible).